api/src/main/resources/workbench.yaml

```diff
@@ -2305,6 +2305,9 @@ definitions:
   UserResponse:
     type: object
     properties:
+      query:
```

Comment:
Hmm.. I'm not sure I like this in principle - does this logically belong in the response? What's the rationale for adding this, but not adding the entirety of the request (present and future) into the response (or doing this on all our endpoints, for that matter)? e.g. imagine we added a second parameter here. I have some other suggestions below.

Reply:
The point of this is for the handler of the response to know what it was that was requested. For example, if there are two responses that arrive out of order, this allows the handler to know the full scope of the response, i.e. `This was searched on and this is what was found`. As it stands now, the response doesn't give the caller the full picture of what was requested. At the end of any series of requests that a caller might make (and whatever order they arrived in), the caller would be able to be able to distinguish between them.

I'm not sure I understand what you're referring to with `the entirety of the request (present and future)`. I don't think we're limited to any number of parameters here.

Reply:
The problem of responses coming back out of order is not unique to this endpoint. Clients have the ability to remember which request corresponds to which response, so there's no need to add API functionality to help them with this.

RE adding parameters, my concern was that the generalized version of this solution involved re-encoding the entire request into the response, which seems like not a great separation of concerns.